### ISSUE TITLE
A bunch of things (VAT, shipping)

### ISSUE THREAD

**Issue by MatteoPiovanelli** [bleroy/Nwazet.Commerce#151]
I realized our development has gone quite a bit ahead with respect to what is on the official repository here. A list of things we have:

 - VAT based on territories (There is an open PR for that #147 )
 - Price services, to replace using the ProductPart's Price property directly (included in the above as it was required there)
 - Query filters, for projections on products (open PR #127)
 - A configurable shipping provider (here: https://github.com/LaserSrl/Nwazet.Commerce/tree/feature/flexibleShippingProviders) that allows creating additional conditions for it to be applied, designed with an approach similar to what has been used for queries in Orchard.Projections.

I can make a PR for the whole thing if you are interested.

**Comment by bleroy** [comment]
Yes, I'm awfully sorry, it was a large PR and I never finished reviewing it. If you can make a new one that is updated, I'll review it, it's a promise. Thanks for your extreme patience.
